### PR TITLE
doc: add upgrade with gitops documents

### PR DESCRIPTION
**Pull request by yangchiu**
#### Which issue(s) this PR fixes:

Issue https://github.com/longhorn/longhorn/issues/8240

#### What this PR does / why we need it:

add upgrade with gitops documents

### CODE REVIEW

**Comment by yangchiu** [comment]
@derekbit For upgrade with gitops, please help review this. Thank you!

File: content/docs/1.6.0/deploy/upgrade/longhorn-manager.md [hunk 1]
```diff
@@ -57,7 +59,46 @@ To upgrade with Helm, run this command:
 helm upgrade longhorn longhorn/longhorn --namespace longhorn-system --version {{< current-version >}}
 ```
 
-On Kubernetes clusters managed by Rancher 2.1 or newer, the steps to upgrade the catalog app `longhorn-system` are the similar to the installation steps.
+#### Upgrade with Fleet
+
+To upgrade with Fleet, update the value of `helm.version` in `fleet.yaml` of your GitOps repository:
```
Comment (jillian-maroket) [inline content/docs/1.6.0/deploy/upgrade/longhorn-manager.md:64]:
```suggestion
Update the value of `helm.version` in the `fleet` YAML file of your GitOps repository.
```

File: content/docs/1.6.0/deploy/upgrade/longhorn-manager.md [hunk 2]
```diff
@@ -57,7 +59,46 @@ To upgrade with Helm, run this command:
...
+helm:
+  repo: https://charts.longhorn.io
+  chart: longhorn
+  version: v1.6.0 # Replace with the Longhorn version you'd like to upgrade to
+  releaseName: longhorn
+```
+
+#### Upgrade with Flux
+
+To upgrade with Flux, update the value of `spec.chart.spec.version` in the `HelmRelease` yaml file of your GitOps repository:
```
Comment (jillian-maroket) [inline content/docs/1.6.0/deploy/upgrade/longhorn-manager.md:76]:
```suggestion
Update the value of `spec.chart.spec.version` in the `HelmRelease` YAML file of your GitOps repository.
```

File: content/docs/1.6.0/deploy/upgrade/longhorn-manager.md [hunk 3]
```diff
@@ -57,7 +59,46 @@ To upgrade with Helm, run this command:
...
+      reconcileStrategy: ChartVersion
+      sourceRef:
+        kind: HelmRepository
+        name: longhorn
+      version: v1.6.0 # Replace with the Longhorn version you'd like to upgrade to
+```
+
+#### Upgrade with Argo CD
+
+To upgrade with Argo CD, update the value of `targetRevision` in the `Application` yaml file of your GitOps repository:
```
Comment (jillian-maroket) [inline content/docs/1.6.0/deploy/upgrade/longhorn-manager.md:92]:
```suggestion
Update the value of `targetRevision` in the `Application` YAML file of your GitOps repository.
```

**Review by jillian-maroket — COMMENTED** [review]
LGTM overall, just a few nits for better readability.

File: content/docs/1.6.0/deploy/upgrade/longhorn-manager.md (hunk 1 above)
Comment (yangchiu) [reply content/docs/1.6.0/deploy/upgrade/longhorn-manager.md]: Refined. Thank you!

File: content/docs/1.6.0/deploy/upgrade/longhorn-manager.md (hunk 2 above)
Comment (yangchiu) [reply content/docs/1.6.0/deploy/upgrade/longhorn-manager.md]: Refined. Thank you!

File: content/docs/1.6.0/deploy/upgrade/longhorn-manager.md (hunk 3 above)
Comment (yangchiu) [reply content/docs/1.6.0/deploy/upgrade/longhorn-manager.md]: Refined. Thank you!

**Review by jillian-maroket — DISMISSED** [review]
LGTM. Thanks!

File: content/docs/1.6.0/deploy/install/install-with-flux.md [hunk 4]
```diff
@@ -42,9 +43,10 @@ weight: 11
     flux create helmrelease longhorn-release \
       --chart=longhorn \
       --source=HelmRepository/longhorn-repo \
-      --chart-version=1.6.0 \
+      --chart-version=v1.6.0 \
```
Comment (derekbit) [inline content/docs/1.6.0/deploy/install/install-with-flux.md:46]: Replace `v1.6.0` with `v{{< current-version >}}`

File: content/docs/1.6.1/deploy/install/install-with-argocd.md [hunk 5]
```diff
@@ -34,6 +34,7 @@ weight: 12
 1. Create the Longhorn Application custom resource.
 
     ```bash
+    kubectl create ns longhorn-system
```
Comment (derekbit) [inline content/docs/1.6.1/deploy/install/install-with-argocd.md:37]: nit:
Or, can we use

```
  syncPolicy:
    syncOptions:
      - CreateNamespace=true
```

File: content/docs/1.6.0/deploy/install/install-with-flux.md (hunk 4 above)
Comment (yangchiu) [reply content/docs/1.6.0/deploy/install/install-with-flux.md]: Refined. Thank you!

File: content/docs/1.6.1/deploy/install/install-with-argocd.md (hunk 5 above)
Comment (yangchiu) [reply content/docs/1.6.1/deploy/install/install-with-argocd.md]: Refined. Thank you!

**Review by derekbit — APPROVED** [review]
LGTM

**Pull request merged**